models/tasks.js

```diff
@@ -193,6 +193,50 @@ const fetchUserTasks = async (username, statuses = [], field, order) => {
   }
 };
 
+const fetchDbForTask = async (skill, level) => {
+  const task = await tasksModel
+    .where("taskLevel.category", "==", skill)
+    .where("taskLevel.level", ">=", level)
+    .where("taskLevel.level", "<=", level + 2)
```

Comment:
Didn't get you here? What are you trying to do?

Reply:
There will be a skill category that will be there in our user object and a level of that skill, we need to return a task based on the skill category and level, suppose you have frontend skill category, level 1, then we want to assign you a task of frontend of level 1 or if it is not available then level 2, and if level 2 task is not available then level 3, so that why we need to query within that range of level and level + 2.

Reply:
Got it.